x/evm/statedb/state_object.go

```diff
@@ -95,7 +96,9 @@ type stateObject struct {
 	address common.Address
 
 	// flags
-	suicided bool
+	dirtyCode      bool
```

Comment:
Im missing a bit of context, what does this flag is used for?

Reply:
can be replaced by `codeDirty()`, removed